ContactManger/src/main/java/Menu.java

```diff
@@ -0,0 +1,27 @@
+import java.util.List;
+import java.util.Scanner;
+
+public class Menu {
+    public static void main(String[] args) {
+        Manager manageContact = new Manager();
```

Comment:
@anamariaTapia You are missing the logic to add new contacts and remove, I was able to see your methods but you don't have a console menu